.bash_profile

```diff
@@ -38,6 +38,11 @@ up()
 # random
 alias reload=". ~/.bash_profile"
 alias today="date +%Y-%m-%d"
+alias snowman="cls && echo '
+  _===_
+ \\(.,.)
```

Comment:
Is the head meant to be off-center?

Reply:
Only one of the two slashes is shown in the console. The other functions as an escape character.